common/app/Panes/Panes.jsx

```diff
@@ -53,31 +55,89 @@ export class Panes extends PureComponent {
       render,
       panes
     } = this.props;
-    return panes.map(({ name, left, right, dividerLeft }) => {
-      const divider = dividerLeft ?
-        (
-          <Divider
-            key={ name + 'divider' }
-            left={ dividerLeft }
-            name={ name }
-          />
-        ) :
-        null;
+    if (panes.length > 0) {
+      return panes.map(({ name, left, right, dividerLeft }) => {
+        const divider = dividerLeft ?
+          (
+            <Divider
+              key={ name + 'divider' }
+              left={ dividerLeft }
+              name={ name }
+            />
+          ) :
+          null;
 
-      return [
-        <Pane
-          key={ name }
-          left={ left }
-          right={ right }
-          >
-          { render(name) }
-        </Pane>,
-        divider
-      ];
-    }).reduce((panes, pane) => panes.concat(pane), [])
-      .filter(Boolean);
+        return [
+          <Pane
+            key={ name }
+            left={ left }
+            right={ right }
+            >
+            { render(name) }
+          </Pane>,
+          divider
+        ];
+      }).reduce((panes, pane) => panes.concat(pane), [])
+        .filter(Boolean);
+    } else {
+      return this.renderQuote();
+    }
+  }
+  renderQuote() {
+    const outerFlexContainer = {
```

Comment:
From what I can see, none of these styles are dynamic. They would be better off in `panes.less` and referenced through class names.

We can raise a separate issue to move the other styles in the render method.